components-rs/telemetry.rs

```diff
@@ -227,54 +231,97 @@ pub unsafe extern "C" fn ddog_sidecar_telemetry_add_integration_log_buffer(
     buffer.buffer.push(SidecarAction::Telemetry(action));
 }
 
+pub struct ShmCache {
+    pub last_updated: Instant,
+    pub integrations: std::collections::HashSet<String>,
+    pub composer_paths: std::collections::HashSet<PathBuf>,
```

Comment:
```suggestion
    pub integrations: HashSet<String>,
    pub composer_paths: HashSet<PathBuf>,
```
Nit, you've anyway `use`'d them at the top of the file.